lib/ui/page/home/widget/dense.dart

```diff
@@ -0,0 +1,32 @@
+// Copyright © 2022-2023 IT ENGINEERING MANAGEMENT INC,
+//                       <https://github.com/team113>
+//
+// This program is free software: you can redistribute it and/or modify it under
+// the terms of the GNU Affero General Public License v3.0 as published by the
+// Free Software Foundation, either version 3 of the License, or (at your
+// option) any later version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+// FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License v3.0 for
+// more details.
+//
+// You should have received a copy of the GNU Affero General Public License v3.0
+// along with this program. If not, see
+// <https://www.gnu.org/licenses/agpl-3.0.html>.
+
+import 'package:flutter/material.dart';
+
+/// Dense [Padding] wrapper.
+class Dense extends StatelessWidget {
+  const Dense(this.child, {super.key});
+
+  /// [Widget] that will be wrapped in padding.
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+        padding: const EdgeInsets.fromLTRB(8, 4, 8, 4), child: child);
```

Comment:
```suggestion
        padding: const EdgeInsets.fromLTRB(8, 4, 8, 4), child: child,);
```

Это уже далеко не первое замечание по форматированию.